presto-hive/src/main/java/io/prestosql/plugin/hive/util/Statistics.java

```diff
@@ -453,11 +454,6 @@ private static OptionalDouble getDoubleValue(ConnectorSession session, Type type
         return block.isNull(0) ? Optional.empty() : Optional.of(LocalDate.ofEpochDay(((SqlDate) type.getObjectValue(session, block, 0)).getDays()));
     }
 
-    private static OptionalLong getTimestampValue(Block block)
```

Comment:
Technically it would be cleaner to remove this in a separate commit

Reply:
You are right. This should be done together with "drop min/max..."